Dockerfile

```diff
@@ -1,15 +1,10 @@
-FROM python:2.7
-
-RUN apt-get update && apt-get install -y wget
+FROM ecarrara/python-gdal
 
 ENV DOCKERIZE_VERSION v0.5.0
```

Comment:
Update this version as well?

Reply:
Yep - it's done with a separate commit https://github.com/ExCiteS/geokey/pull/404/commits/3a5226fbec5115c84feb5592d87aea7cdaf43d38